src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -32,6 +31,8 @@
     timeout=settings.SEER_ANOMALY_DETECTION_TIMEOUT,
 )
 
+NOT_ENOUGH_DATA = "Fewer than seven days of historical data available"
```

Comment:
nit: can we update this to be `NOT_ENOUGH_DATA_MSG` to make it more explicit from the status?

Reply:
We don't use it anymore, so I'll just get rid of it